runtime/dom-particle.js

```diff
@@ -108,13 +108,24 @@ export class DomParticle extends XenStateMixin(Particle) {
   }
   async setHandles(handles) {
     this.handles = handles;
-    let config = this.config;
-    this.when([new HandleChanges(handles, config.handles, 'change')], async () => {
-      await this._handlesToProps(handles, config);
-    });
+
+    // this.config isn't a static property; not sure how important it is to avoid re-generating
+    // it in the onHandle* functions
+    this._configSnapshot = this.config;
```

Comment:
the `config` getter is an adapter to `this.spec` (the particle spec). the one thing that `_handlesToProps` needs it the input handle names (ie `this.spec.inputs.map(i => i.name)`).
so IMO we should get rid of it altogether, but this is out of scope of this PR.  but i don't think you need to snapshot it, i'd pass `this.config` directly.

Reply:
Thanks for the info. Is is possible for the spec field to change over the lifetime of DomParticle? (I would expect no, but wasn't sure; hence the snapshot.)